### DISCUSSION TITLE
看不到报错日志

### DISCUSSION THREAD

**Discussion by gochendong** [apache/dolphinscheduler#7384] [Q&A]
cannot find my error log![image](https://user-images.githubusercontent.com/39932550/145916423-a4ec212a-91d4-42f2-80bb-95a01c76eeaa.png)

**Comment by xiaoyeyisheng** [comment]
tail -f dolphinScheduler/logs/*  Watch for errors in the log

**Comment by Tandoy** [comment]
First confirm whether the logger server is started normally,and then execute the following command to view the error log
`grep failover dolphinScheduler/logs/*`
btw,please describe your question in English
